package.json

```diff
@@ -1,37 +1,37 @@
 {
-  "name": "@lifeomic/eslint-config-standards",
+  "name": "@lifeomic/eslint-config",
```

Comment:
I think I'd vote for not renaming, if others are alright with it. I think `@lifeomic/standards` is a nice reference syntax, and that way we can avoid:
- renaming the repo
- having two npm packages published

Reply:
I'm okay with it also, I just like the idea of just using the namespsce as well.